uncrustify.cfg

```diff
@@ -40,12 +42,14 @@ nl_namespace_brace		= add
 mod_full_brace_if		= remove
 mod_full_brace_if_chain	= true
 mod_full_brace_do		= remove
-mod_full_brace_while		= remove
+mod_full_brace_while	= remove
 mod_full_brace_for		= remove
-mod_full_brace_if_chain		= true
+mod_full_brace_if_chain	= true
 mod_full_brace_nl		= 3
-mod_full_paren_if_bool		= true
+mod_full_paren_if_bool	= true
```

Comment:
@evsh:  This line is definitely not spaced using a tab.

Reply:
No, it is.